src/transformers/processing_utils.py

```diff
@@ -226,7 +455,19 @@ def from_pretrained(
             kwargs["token"] = token
 
         args = cls._get_arguments_from_pretrained(pretrained_model_name_or_path, **kwargs)
-        return cls(*args)
+
+        # Existing processors on the Hub created before #27761 being merged don't have `processor_config.json` (if not
+        # updated afterward), and we need to keep `from_pretrained` work. So here it fallbacks to the empty dict.
+        # However, for models added in the future, we won't get the expected error if this file is missing.
+        try:
+            processor_dict, kwargs = cls.get_processor_dict(pretrained_model_name_or_path, **kwargs)
+        except EnvironmentError as e:
+            if "does not appear to have a file named processor_config.json." in str(e):
+                processor_dict, kwargs = {}, kwargs
```

Comment:
this is a bit brittle but alright

Reply:
Yeah, this is not ideal but I don't see better way.